src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriverDevicePin.Linux.cs

```diff
@@ -42,6 +56,7 @@ public void Dispose()
                 FileDescriptor = -1;
             }
 
+            ActiveEdges = PinEventTypes.None;
```

Comment:
Technically this is not required. This s just a regular enum, so doesn't need resetting to original value on dispose.

Reply:
I know. This is just to make sure the state of the class is consistent. Just in case somebody reuses the disposed instance.